sdk/iot/Azure.Iot.Hub.Service/src/API Design.md

```diff
@@ -64,8 +64,43 @@ APIs for using IotHub v2 jobs
 
 <details><summary><b>Messages</b></summary>
 Feedback messages, sending cloud to device messages (missing from current swagger), and purging cloud to device message queue
-```csharp
 
+```csharp
+public class Messages
+{
+    /// <summary>
+    /// Retrieve feedback notification for cloud to device messages.
+    /// </summary>
+    /// <param name="cancellationToken">The cancellation token.</param>
+    /// <returns>TODO: Swagger defines the response as the Http response returned from the service (and not the feedback message).</returns>
+    public virtual async Task<Response> GetCloudToDeviceMessageFeedbackAsync(CancellationToken cancellationToken = default) { }
```

Comment:
Is it better to call it **Receive**CloudToDeviceMessage**s**FeedbackAsync instead?

Reply:
I can get on board with that; let's discuss this in today's meeting.